inspirehep/modules/authors/tasks.py

```diff
@@ -164,14 +165,14 @@ def curation_ticket_context(user, obj):
     if obj.data.get("bai"):
         bai = "[{}]".format(obj.data.get("bai"))
     subject = "Curation needed for author {0} {1}".format(
-        obj.data.get("name").get("preferred_name"),
-        bai
+        obj.data.get("name").get("preferred_name"), bai
     )
     return dict(
         email=user.email,
         object=obj,
         recid=recid,
         subject=subject,
         record_url=record_url,
-        user_comment=obj.extra_data.get('formdata', {}).get('extra_comments', ''),
+        user_comment=obj.extra_data.get('formdata', {}
+                                        ).get('extra_comments', ''),
```

Comment:
strange that sometimes it decides to indent with 4 spaces, and sometimes it just goes all the way to the opening char. I prefer 4 spaces so it always matches the logical nested level.